pkg/router/functionHandler.go

```diff
@@ -440,8 +478,70 @@ func (fh *functionHandler) tapService(fn *fv1.Function, serviceURL *url.URL) {
 	fh.executor.TapService(fn.ObjectMeta, fn.Spec.InvokeStrategy.ExecutionStrategy.ExecutorType, *serviceURL)
 }
 
+func (fh functionHandler) parseStickinessCookie(request *http.Request) (*stickinessCookie, *http.Cookie, error) {
+	var stickinessCookie stickinessCookie
+	cookie, err := request.Cookie(STICKINESS_COOKIE)
+	if err != nil {
+		fh.logger.Debug("failed to get stickiness cookie value", zap.Error(err))
```

Comment:
Are this "errors" not important enough to be Errors? Should we keep them with Debug?

Reply:
This one will happen on every new client / connection so would keep as debug

Reply:
If it fails to base64 decode / json unmarshal then that should potentially be an error however I don't think we can log the cookie contents in this case (potential security risk)

Reply:
I have increased the relevant log levels to WARN and removed the cookie contents from the log, let me know if you agree